CHANGELOG.md

```diff
@@ -26,6 +26,9 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/report/compare/release-3.12.0...master
 
+### Deprecated
+- Mark `RelativeNonparametricStabilityJudge` as deprecated.
```

Comment:
Do we release minor versions when we deprecate? I remember some discussion about it, however I can't find a decisive information in the CHANGELOG.md about it.

Reply:
nvm, I found it: https://semver.org/#how-should-i-handle-deprecating-functionality

Releasing the minor